test/Microsoft.DotNet.ProjectModel.Tests/PackageDependencyProviderTests.cs

```diff
@@ -8,16 +10,46 @@
 using NuGet.Frameworks;
 using NuGet.Versioning;
 using Xunit;
-using System.IO;
 
 namespace Microsoft.DotNet.ProjectModel.Tests
 {
     public class PackageDependencyProviderTests : TestBase
     {
         [Fact]
+        public void GetDescriptionShouldLeavePackageLibraryPath()
```

Comment:
ShouldLeavePackageLibraryPath? Alone? Should include PackageLibraryPath? This is named strangely.